### PR TITLE
Added column sorting to Add from HDB dialogs in ProDe

### PR DESCRIPTION
**Pull request by alexhrv**

### CODE REVIEW

**Comment by alexhrv** [comment]
Not sure if the ProDeConstants.cc is the correct place to store arrow images

**Comment by linehill** [comment]
I think, the good place is to put icons under src/applibs/wxToolit/ where the common wxWidgets utilities are.

**Comment by pjaaskel** [comment]
Is there a reason this image is in C. Why not put a bitmap of it under data/icons?

**Comment by alexhrv** [comment]
No special reason, I took example image from OperationPropertyDialog.cc file in OSEd. Putting it under data/icons sounds good to me

File: tce/src/procgen/ProDe/AddFUFromHDBDialog.cc [hunk 1]
```diff
@@ -86,18 +129,28 @@ AddFUFromHDBDialog::AddFUFromHDBDialog(
...
 
     list_ = dynamic_cast<wxListCtrl*>(FindWindow(ID_LIST));
 
-    list_->InsertColumn(0, _T("Latency"));
-    list_->InsertColumn(1, _T("Operations"));
-    list_->InsertColumn(2, _T("Impls"));
-    list_->InsertColumn(3, _T("HDB ID"));
-    list_->InsertColumn(4, _T("HDB"));
+    list_->InsertColumn(0, _T("Latency"), wxLIST_FORMAT_LEFT, wxLIST_AUTOSIZE);
+    list_->InsertColumn(1, _T("Operations"), wxLIST_FORMAT_LEFT, wxLIST_AUTOSIZE);
```
Comment (linehill) [inline tce/src/procgen/ProDe/AddFUFromHDBDialog.cc]: Over 80 character row here

**Pull request merged**